ext/RMagick/extconf.rb

```diff
@@ -186,6 +159,27 @@ def exit_failure(msg)
       exit(1)
     end
 
+    def determine_imagemagick_package
+      exit_failure "Can't install RMagick #{RMAGICK_VERS}. Can't find pkg-config in #{ENV['PATH']}\n" unless find_executable('pkg-config')
+
+      packages = `pkg-config --list-all`.scan(/(ImageMagick\-6[\.A-Z0-9]+) .*/)
```

Comment:
I wonder if it would be worth tacking a `.flatten` on the end here to avoid the nesting below.

Reply:
Makes it more clear, thanks for the hint.